packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -0,0 +1,1166 @@
+
+import docs from 'docs:@react-aria/combobox';
+import collectionsDocs from 'docs:@react-types/shared/src/collections.d.ts';
+import {FunctionAPI, HeaderInfo, InterfaceType, renderHTMLfromMarkdown, TypeContext, TypeLink} from '@react-spectrum/docs';
+import i18nDocs from 'docs:@react-aria/i18n';
+import listboxDocs from 'docs:@react-aria/listbox';
+import overlaysDocs from 'docs:@react-aria/overlays';
+import packageData from '@react-aria/combobox/package.json';
+import selectionDocs from 'docs:@react-stately/selection';
+import statelyDocs from 'docs:@react-stately/combobox';
+
+
+```jsx import
+import {useComboBox} from '@react-aria/combobox';
+```
+
+---
+category: Pickers
+keywords: [combobox, aria]
+---
+
+# useComboBox
+
+<p>{renderHTMLfromMarkdown(docs.exports.useComboBox.description)}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useComboBox']} />
+
+## API
+
+<FunctionAPI function={docs.exports.useComboBox} links={docs.links} />
+
+## Features
+
+There is no native element to implement a ComboBox in HTML. `useComboBox` helps achieve accessible ComboBox components that can
+be styled as needed.
+
+* Exposed to assistive technology as a ComboBox
+* Support for selecting a single option
+* Support for disabled options
+* Support for sections
+* Support for custom user input values
+* Labeling support for accessibility
+* Required and invalid states exposed to assistive technology via ARIA
+* Support for mouse, touch, and keyboard interactions
+* Keyboard support for opening the ComboBox menu using the arrow keys, including automatically focusing
+  the first or last item accordingly
+* Virtual focus management for ComboBox menu option navigation
+* VoiceOver announcement enhancements for option focusing, filtering, and selection
+
+## Anatomy
+
+A ComboBox consists of a label, an input which displays the current value, a listbox popup, and a button
+used to toggle the listbox popup open state. Users can type within the input to filter the available options
+within the listbox popup. The listbox popup may be opened by a variety of input field interactions specified
+by the `menuTrigger` prop provided to `useComboBox`, or by clicking or touching the ComboBox trigger button. `useComboBox` handles exposing
+the correct ARIA attributes for accessibility for each of the components comprising the ComboBox. It should be combined
+with [useListBox](useListBox.html), which handles the implementation of the popup listbox,
+and [useButton](useButton.html) which handles the button press handlers returned by `useComboBox`.
+
+`useComboBox` returns props that you should spread onto the appropriate elements:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useComboBox.return.id].properties} />
+</TypeContext.Provider>
+
+State is managed by the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useComboBoxState} /> hook from `@react-stately/select`.
```

Comment:
Why is this a TypeLink instead directly to the stately docs page?

Reply:
Follows the pattern set up by previous stately doc pages. The stately pages are typically pretty bare, so having this TypeLink is sufficient since it shows the stately hook's api.